packages/composables/src/useCountry/index.ts

```diff
@@ -1,55 +1,54 @@
-import {
-  sharedRef,
-  Logger,
-  configureFactoryParams
-} from '@vue-storefront/core';
+import { sharedRef, useVSFContext } from '@vue-storefront/core';
 import { computed } from '@vue/composition-api';
+import { UseCountry } from '../types';
 
-const useCountryFactory = (factoryParams) => {
-  return function useCountry() {
-    const loading = sharedRef(false, 'useCountry-loading');
-    const countries = sharedRef(null, 'useCountry-countries');
-    const error = sharedRef({ load: null }, 'useCountry-error');
+export default function useCountry(): UseCountry {
+  const context = useVSFContext();
 
-    const _factoryParams = configureFactoryParams(factoryParams);
+  const countries = sharedRef(null, 'useCountry-countries');
+  const states = sharedRef(null, 'useCountry-states');
```

Comment:
I'm not sure whether this should be a sharedRef like that.
I can imagine that in some cases, someone may need to show both forms for shipping and billing on the same page. 

https://github.com/vuestorefront/vue-storefront/blob/9fcfb30e168a104b0629c83723f57c1a5b012e6a/packages/core/core/src/factories/useProductFactory.ts#L15-L15

Take a look here - the useProduct takes an optional `id` param, that's then used to setup sharedRefs.
This way we could enable the dev to call `useCountry('shipping')` and `useCountry('billing')`, with the states list being isolated. The countries ref is fine to be global, as it's not parameterized :)